op.c

```diff
@@ -12060,8 +12060,16 @@ Constructs, checks, and returns a glob reference op.
 OP *
 Perl_newGVREF(pTHX_ I32 type, OP *o)
 {
-    if (type == OP_MAPSTART || type == OP_GREPSTART || type == OP_SORT)
-        return newUNOP(OP_NULL, 0, o);
+    switch(type) {
+        /* The thing that looks like a GVREF at the start of these operators
+         * isn't really */
+        case OP_MAPSTART:
+        case OP_GREPSTART:
+        case OP_SORT:
+        case OP_ANYSTART:
+        case OP_ALLSTART:
```

Comment:
Can we still get here? The new `BLKLSTOP` grammar rule doesn't call `newGVREF()`

Reply:
It seems so. I put an `assert(0)` in there and the `t/op/any_all.t` file immediately provokes it at compiletime.

Reply:
The FUNC rule calls it.